RELEASE.md

```diff
@@ -0,0 +1,243 @@
+
+```
+make pre-release
+```
+
+Commit the changes on your release branch and push them:
+
+```bash
+git checkout v{major}.{minor}-release
+git add setup.py src
+git commit -m "Release: v{VERSION}"
+git push -u origin v{major}.{minor}-release
+```
+
+### Step 4: Make sure all tests pass
+
+Your CI should run on the release branch (if it doesn’t, you should make it do that 😅). Make sure all tests are green before proceeding to the next step.
+
+### Step 5: Add a tag for your release
+
+A tag will flag the exact commit associated to your release (and be easier to remember than the commit hash!). The tag should be `v<VERSION>` so for instance `v4.12.0` . This is important for the documentation.
+
+Here is how you can create and push your tag:
+
+```bash
+git tag v<VERSION> -m 'Adds tag v<VERSION> for PyPI'
+git push --tags origin v{major}.{minor}-release
+```
+
+### Step 6: Create the wheels for your release
+
+This is the stuff you will upload on PyPI and that everyone will then download each time they pip install your package.
+
+Clean previous builds by running:
+
+```
+make wheels_clean
+```
+
+Then run:
+
+```bash
+make wheels
+```
+
+This will create two folders, `build` and a `dist` with the new versions of your package.
+
+### Step 7: Upload your package on PyPI test
+
+**DO NOT SKIP THIS STEP!**
+
+This is the most important check before actually releasing your package in the wild. Upload the package on PyPI test and check you can properly install it.
+
+To upload it:
+
+```bash
+make pypi_test_upload
+```
+
+You will be prompted for your username and password.
+
+Then in a fresh environment containing all dependencies you need, try to install your new package from the pypi test server
+
+```bash
+make pypi_test_install
+```
+
+If everything works, you should get the following message:
+
+```
+🚀 Successfully installed setfit from test.pypi.org
+```
+
+### Step 8: Publish on PyPI
+
+This cannot be undone if you messed up, so make sure you have run Step 7!
+
+Once you’re fully ready, upload your package on PyPI:
+
+```bash
+make pypi_upload
+```
+
+You will be prompted for your username and password.
+
+### Step 9: Publish your release notes
+
+Go back to the draft you did at step 2 ([https://github.com/huggingface/setfit/releases](https://github.com/huggingface/setfit/releases) with project_name replaced by the package you are working on) and add the tag you created at step 4 to your release before publishing your release notes
+
+### Step 10: Bump the dev version on the main branch
+
+You’re almost done! Just go back to the `main` branch and change the dev version in the `setup.py` and the main `__init__.py` to the new version you’re developing, for instance `4.13.0.dev0` if just released `4.12.0`
+
+### Patch releases
+
+Follow this guide when you are releasing a patch to fix a bug introduced in a release. For minor or major releases go [here](https://www.notion.so/Make-a-release-81fbe41ad0e24657a3126dfc7fd1c4b6).
```

Comment:
```suggestion
Follow this guide when you are releasing a patch to fix a bug introduced in a release. For minor or major releases go [here](#minor-or-major-releases).
```
I think this should work, but I'm not sure.

Reply:
Nice catch!